From 4a4f13aa52b7e36aa381c0a6b66a09e24cf1908c Mon Sep 17 00:00:00 2001
From: Subhajit Paul <subhajit_paul@ti.com>
Date: Thu, 11 Jul 2019 10:58:22 +0530
Subject: [PATCH] turn off msmc cache for psdkra : cache size 0

Signed-off-by: Subhajit Paul <subhajit_paul@ti.com>
---
 soc/j721e/evm/board-cfg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/soc/j721e/evm/board-cfg.c b/soc/j721e/evm/board-cfg.c
index b4b2cb1..9acf3d8 100644
--- a/soc/j721e/evm/board-cfg.c
+++ b/soc/j721e/evm/board-cfg.c
@@ -69,7 +69,7 @@ const struct boardcfg j721e_boardcfg_data = {
 			.magic = BOARDCFG_MSMC_MAGIC_NUM,
 			.size = sizeof(struct boardcfg_msmc),
 		},
-		.msmc_cache_size = 0x10,
+		.msmc_cache_size = 0x0,
 	},
 
 	/* boardcfg_dbg_cfg */
-- 
2.7.4

