From cde8cb068775961b4343e20279b8a9ee36bc3600 Mon Sep 17 00:00:00 2001
From: Dharma B <dharma.b@ti.com>
Date: Tue, 30 Nov 2021 19:16:18 +0530
Subject: [PATCH 08/17] ti_config_fragments/connectivity.cfg: enable pps bc
 support

Signed-off-by: Dharma B <dharma.b@ti.com>
---
 ti_config_fragments/connectivity.cfg | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/ti_config_fragments/connectivity.cfg b/ti_config_fragments/connectivity.cfg
index 036cdf776060..3e8dbfc05e48 100644
--- a/ti_config_fragments/connectivity.cfg
+++ b/ti_config_fragments/connectivity.cfg
@@ -510,5 +510,8 @@ CONFIG_SERIAL_8250_PRUSS=m
 # MUX Drivers
 CONFIG_MUX_GPIO=y
 
+CONFIG_TI_1PPS_DM_TIMER=y
+CONFIG_TI_PTP_BC=y
+
 # PRU Soft UART driver
 CONFIG_SERIAL_PRU_SWUART=m
-- 
2.17.0

