From a97440bb7f8f186e2e51fa3eda21002cc7ab0380 Mon Sep 17 00:00:00 2001
From: Grygorii Strashko <grygorii.strashko@ti.com>
Date: Mon, 8 Feb 2021 17:29:18 +0200
Subject: [PATCH 24/35] cpts: latch: drop dbg from cpts_1pps_latch_interrupt

Signed-off-by: Grygorii Strashko <grygorii.strashko@ti.com>
---
 drivers/net/ethernet/ti/cpts.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/net/ethernet/ti/cpts.c b/drivers/net/ethernet/ti/cpts.c
index 27cc3b06c792..b56a4f66e1c8 100644
--- a/drivers/net/ethernet/ti/cpts.c
+++ b/drivers/net/ethernet/ti/cpts.c
@@ -2087,7 +2087,6 @@ static irqreturn_t cpts_1pps_tmr_interrupt(int irq, void *dev_id)
 	return IRQ_HANDLED;
 }
 
-static int latch_cnt;
 static irqreturn_t cpts_1pps_latch_interrupt(int irq, void *dev_id)
 {
 	struct cpts *cpts = (struct cpts *)dev_id;
@@ -2096,11 +2095,6 @@ static irqreturn_t cpts_1pps_latch_interrupt(int irq, void *dev_id)
 
 	cpts_latch_proc(cpts, READ_TCAP(cpts->odt2));
 
-	if (latch_cnt <= 100)
-		latch_cnt++;
-	if ((latch_cnt % 10) == 0)
-		dev_dbg(cpts->dev, "%s %d\n", __func__, latch_cnt);
-
 	return IRQ_HANDLED;
 }
 #endif
-- 
2.17.1

