From 4374cd9f58b2d4e0e5957b8305dfff315b1aaeb6 Mon Sep 17 00:00:00 2001
From: Nikhil Devshatwar <nikhil.nd@ti.com>
Date: Fri, 7 Feb 2020 18:15:20 +0530
Subject: [PATCH] arm: dts: k3-j721e: Do not use lockstep mode for Main R5FSS1

Signed-off-by: Nikhil Devshatwar <nikhil.nd@ti.com>
---
 arch/arm/dts/k3-j721e-main.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/dts/k3-j721e-main.dtsi b/arch/arm/dts/k3-j721e-main.dtsi
index 94d70f2dcb..728aec6dbd 100644
--- a/arch/arm/dts/k3-j721e-main.dtsi
+++ b/arch/arm/dts/k3-j721e-main.dtsi
@@ -436,7 +436,7 @@
 
 	main_r5fss1: r5fss@5e00000 {
 		compatible = "ti,j721e-r5fss";
-		lockstep-mode = <1>;
+		lockstep-mode = <0>;
 		#address-cells = <1>;
 		#size-cells = <1>;
 		ranges = <0x5e00000 0x00 0x5e00000 0x20000>,
-- 
2.17.1

