--- a/tests/python/integration/load_and_run_tvm_model.py	2020-11-15 21:37:24.549579751 -0600
+++ b/tests/python/integration/load_and_run_tvm_model_new.py	2020-11-15 21:37:25.189586194 -0600
@@ -109,7 +109,7 @@
         get_models(model_name, arch, kind='tvm')
     test_resnet()
     test_mobilenet_v1_0_75_224_quant()
-    test_mobilenet_v1_0_75_224_quant_wrong_input_type()
+   #test_mobilenet_v1_0_75_224_quant_wrong_input_type()
     test_multi_input_multi_output()
     test_assign_op()
     print('All tests passed!')
