From 91689cfefd2f9b039e7d596bd535973d9486db59 Mon Sep 17 00:00:00 2001
From: Subhajit Paul <subhajit_paul@ti.com>
Date: Sat, 10 Oct 2020 00:38:40 +0530
Subject: [PATCH] [not-upstream][yocto] run without REST

Signed-off-by: Subhajit Paul <subhajit_paul@ti.com>
---
 python/dlr/api.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/dlr/api.py b/python/dlr/api.py
index ee9f4a4..c146dc2 100644
--- a/python/dlr/api.py
+++ b/python/dlr/api.py
@@ -6,7 +6,7 @@ import glob
 import os
 import logging
 from .neologger import create_logger
-from .counter import call_phone_home
+#from .counter import call_phone_home
 
 
 # Interface
@@ -58,7 +58,7 @@ def _is_module_found(name):
 # Wrapper class
 class DLRModel(IDLRModel):
     
-    @call_phone_home
+    #@call_phone_home
     def __init__(self, model_path, dev_type=None, dev_id=None, error_log_file=None, use_default_dlr=False):
         """
         Load a Neo-compiled model.
-- 
2.17.1

