From 3762c9bd51f03f1b90782ac6fb74383c295018bc Mon Sep 17 00:00:00 2001
From: Lucas Weaver <l-weaver@ti.com>
Date: Thu, 4 Nov 2021 10:08:14 -0500
Subject: [PATCH] HACK: Increasing MBOX length

---
 include/linux/mailbox_controller.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/mailbox_controller.h b/include/linux/mailbox_controller.h
index 36d6ce673..fe3389e07 100644
--- a/include/linux/mailbox_controller.h
+++ b/include/linux/mailbox_controller.h
@@ -99,7 +99,7 @@ struct mbox_controller {
  * REVISIT: If too many platforms see the "Try increasing MBOX_TX_QUEUE_LEN"
  * print, it needs to be taken from config option or somesuch.
  */
-#define MBOX_TX_QUEUE_LEN	20
+#define MBOX_TX_QUEUE_LEN	40
 
 /**
  * struct mbox_chan - s/w representation of a communication chan
-- 
2.17.1

