From 95eafe0c3fb697ef5bc022583f5b356c61d26e16 Mon Sep 17 00:00:00 2001
From: Grygorii Strashko <grygorii.strashko@ti.com>
Date: Mon, 21 Dec 2020 12:46:52 +0200
Subject: [PATCH 03/15] util: add SIGHUP handling

Signed-off-by: Grygorii Strashko <grygorii.strashko@ti.com>
---
 util.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/util.c b/util.c
index 3cc615a..bca0631 100644
--- a/util.c
+++ b/util.c
@@ -575,6 +575,9 @@ int handle_term_signals(void)
 		fprintf(stderr, "cannot handle SIGTERM\n");
 		return -1;
 	}
+	if (SIG_ERR == signal(SIGHUP, handle_int_quit_term)) {
+		fprintf(stderr, "cannot handle SIGHUP\n");
+	}
 	return 0;
 }
 
-- 
2.17.1

