From a82104236b67ca3c03d9ea77d79b3d33e4b5f635 Mon Sep 17 00:00:00 2001
From: Grygorii Strashko <grygorii.strashko@ti.com>
Date: Tue, 9 Feb 2021 20:16:05 +0200
Subject: [PATCH 09/15] phc2sys.c always kill servo when switch to master for
 BC

Signed-off-by: Grygorii Strashko <grygorii.strashko@ti.com>
---
 phc2sys.c | 16 +++++++---------
 1 file changed, 7 insertions(+), 9 deletions(-)

diff --git a/phc2sys.c b/phc2sys.c
index 78bde32..fc7760d 100644
--- a/phc2sys.c
+++ b/phc2sys.c
@@ -358,19 +358,17 @@ static void clock_reinit(struct node *node, struct clock *clock, int new_state)
 			clock->clkid = clkid;
 			clock->phc_index = phc_index;
 
-			if (clock->servo) {
-				servo_destroy(clock->servo);
-				clock->servo = NULL;
-			}
-
 			phc_switched = 1;
 		}
 	}
 
-	if (new_state == PS_MASTER || phc_switched) {
-		if (clock->servo)
-			servo_reset(clock->servo);
-		clock->servo_state = SERVO_UNLOCKED;
+	if (new_state == PS_MASTER || new_state == PS_PRE_MASTER ||
+	    phc_switched) {
+		if (clock->servo) {
+			servo_destroy(clock->servo);
+			clock->servo = NULL;
+			clock->servo_state = SERVO_UNLOCKED;
+		}
 
 		if (clock->offset_stats) {
 			stats_reset(clock->offset_stats);
-- 
2.17.1

