diff --git a/arch/arm/dts/k3-j721e-common-proc-board-u-boot.dtsi b/arch/arm/dts/k3-j721e-common-proc-board-u-boot.dtsi
index cfb39325e9..33adc515fb 100644
--- a/arch/arm/dts/k3-j721e-common-proc-board-u-boot.dtsi
+++ b/arch/arm/dts/k3-j721e-common-proc-board-u-boot.dtsi
@@ -62,6 +62,8 @@
 
 &k3_clks {
 	u-boot,dm-spl;
+	assigned-clocks = <&k3_clks 61 1>;
+	assigned-clock-rates = <200000000>;
 };
 
 &k3_reset {
