From 48869f5cfb02117fbb57264cf50c191129d94a51 Mon Sep 17 00:00:00 2001
From: Subhajit Paul <subhajit_paul@ti.com>
Date: Fri, 8 Jan 2021 19:20:09 +0530
Subject: [PATCH 3/3] add graphviz to install dependency

Signed-off-by: Subhajit Paul <subhajit_paul@ti.com>
---
 python/setup.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/python/setup.py b/python/setup.py
index 402d99382..ea280a94a 100644
--- a/python/setup.py
+++ b/python/setup.py
@@ -163,6 +163,7 @@ setup(
         "attrs",
         "psutil",
         "typed_ast",
+        "graphviz",
     ],
     extras_require={
         "test": ["pillow<7", "matplotlib"],
-- 
2.17.1

