From 49859128e942ea8a0fd5cf2a9761d7ff0c950fa7 Mon Sep 17 00:00:00 2001
From: Aniket Limaye <a-limaye@ti.com>
Date: Wed, 18 Aug 2021 00:04:38 +0530
Subject: [PATCH] Added '--host=aarch64-linux' to ThirdpartyToolchain.cmake

Signed-off-by: Aniket Limaye <a-limaye@ti.com>
---
 cmake_modules/ThirdpartyToolchain.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmake_modules/ThirdpartyToolchain.cmake b/cmake_modules/ThirdpartyToolchain.cmake
index e6852d9..3191b2d 100644
--- a/cmake_modules/ThirdpartyToolchain.cmake
+++ b/cmake_modules/ThirdpartyToolchain.cmake
@@ -1515,6 +1515,7 @@ if(ARROW_JEMALLOC)
        "--disable-libdl"
        # See https://github.com/jemalloc/jemalloc/issues/1237
        "--disable-initial-exec-tls"
+       "--host=aarch64-linux"
        ${EP_LOG_OPTIONS})
   set(JEMALLOC_BUILD_COMMAND ${MAKE} ${MAKE_BUILD_ARGS})
   if(CMAKE_OSX_SYSROOT)
-- 
2.17.1

